PWGCF/FemtoDream/FemtoDreamV0Selection.h

```diff
@@ -264,6 +280,13 @@ bool FemtoDreamV0Selection::isSelectedMinimal(C const& col, V const& v0, T const
   const float dcaDaughv0 = v0.dcaV0daughters();
   const float cpav0 = v0.v0cosPA(col.posX(), col.posY(), col.posZ());
 
+  const float invMassLambda = v0.mLambda();
+  const float invMassAntiLambda = v0.mAntiLambda();
+
+  if ((invMassLambda < fInvMassLowLimit or invMassLambda > fInvMassUpLimit) and (invMassAntiLambda < fInvMassLowLimit or invMassAntiLambda > fInvMassUpLimit)) {
```

Comment:
So now you reject when both Lambda AND ALambda are out of the limits but you accept that the Lambda is within and the ALambda is outside the limits or the other way around. Right?

Reply:
Correct